README.md

```diff
@@ -1,11 +1,61 @@
-# KEVM Ganache-CLI fork
+KEVM Ganache-CLI Fork
+=====================
+
+Installing KEVM
+---------------
 
-## Installing KEVM
 In order to run `kevm-ganache-cli`, you will need to install `kevm-vm`.
-We currently support [Ubuntu](#Ubuntu), [Arch](#Arch), and [MacOS](#MacOS).
+We currently support Ubuntu Bionic (18.04) and Arch Linux (MacOS coming soon).
+Here a specific GitHub release is used, feel free to replace the URL with a newer release from <https://github.com/kframework/evm-semantics/releases>.
+
+### Ubuntu Bionic (18.04) - fresh VM
```

Comment:
Also, changing these titles will change the anchor links. It might be better to keep the title as `Ubuntu` and add the details below (to keep the url as `.../readme#ubuntu` instead of `.../readme#ubuntu-bionic-18-04---fresh-vm` or whatever it would end up being.

Reply:
Done